src/client/js/Widgets/TreeBrowser/TreeBrowserWidget.js

```diff
@@ -364,10 +365,12 @@ define(['js/logger',
             isConnection: objDescriptor.isConnection,
             isAbstract: objDescriptor.isAbstract,
             isLibrary: objDescriptor.isLibrary,
+            isLibraryRoot: objDescriptor.isLibraryRoot,
             libraryInfo: objDescriptor.libraryInfo,
             metaType: objDescriptor.metaType
         }, beforeNode);
 
+        this.sortChildren(parentNode, false);
```

Comment:
Doesn't this mean that it sorts the children even though there wasn't any changes?